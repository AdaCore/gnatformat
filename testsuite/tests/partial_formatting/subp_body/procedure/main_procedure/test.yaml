description: SubpBody formatting with partial selection a main procedure.
driver: partial_gnatformat
args:
  [
    "main.adb",
    "-SL 1",
    "-EL 10",
  ]
status_code: 1