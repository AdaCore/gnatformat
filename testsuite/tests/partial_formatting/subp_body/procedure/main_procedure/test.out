<CompilationUnit main.adb:1:1-10:10>
main.adb(<SubpBody ["main"] main.adb:1:1-10:10>) - 1:1-10:10
^
procedure main
is
begin

   declare
      A : Integer;
   begin
      null;
   end;

end main;$

